theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
Indeed, it's generally about array of strides or dimensions. But are we sure that users always work with at most 32 dimensions ^^ ? I think we can't do that assumption, unfortunately.